package.json

```diff
@@ -110,6 +110,7 @@
         "not Android > 0"
     ],
     "overrides": {
-        "aria-query": "~5.0.0"
+        "aria-query": "~5.0.0",
+        "@web/dev-server-core": "0.7.1"
```

Comment:
https://github.com/modernweb-dev/web/issues/2772#issuecomment-2301837407

Newer versions of this transitive dependencies make the output of failing tests very hard to debug.